modules/developer-cli-odo-sample-applications.adoc

```diff
@@ -93,6 +95,9 @@ $ odo create java test3 --binary target/*.jar
 $ odo push
 ----
 
+
+//Commenting out as it doesn't work for now. https://github.com/openshift/odo/issues/4623
```

Comment:
IBM SG: 
`as adverb
Do not use to mean "because" or "while".
Example: Write "Because you created the file, you can delete it", not "As you created the file, you can delete
it".  `
LOL - just teasing you.